### PR TITLE
Common readout defintion

### PR DESCRIPTION
**Pull request by jhrdinka**
Addresses #247 .

As agreed in the FCCSW meeting this merge request defines the readout relevant IDs globally in  to guarantee consistency.

Changes proposed in this pull-request:

- This merge request just copies all readout IDs of `FCChhBaseline1`  into  `FCChh_DectDimensions.xml` and does not change any content

This merge request is based on #252 (tracker update) and should be merged afterwards.

### CODE REVIEW

File: Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml [hunk 1]
```diff
@@ -143,18 +123,59 @@
...
+    <!-- Hcal -->
+    <constant name="BarHCal_ReadoutID" type="string" value="system:4,module:7,row:9,layer:5,tile:2,eta:1,phi:10"/>
+    <constant name="BarHCal_Readout_phietaID" type="string" value="system:4,module:7,row:9,layer:5,tile:2,eta:9,phi:10"/>
+    <!-- Ext barrel Hcal -->
+    <constant name="ExtBarHCal_ReadoutID" type="string" value="system:4,type:2,module:9,row:8,layer:4,tile:2,eta:1,phi:10"/>
+    <constant name="ExtBarHCal_Readout_phietaID" type="string" value="system:4,type:2,module:9,row:8,layer:4,tile:2,eta:10,phi:10"/>
+    <!-- Calo endcaps and fwd -->
+    <constant name="CalEndCapID" type="string" value="system:4,subsystem:1,type:3,subtype:3,layer:8,eta:10,phi:10"/>
+    <constant name="CalFwdID" type="string" value="system:4,subsystem:1,type:3,subtype:3,layer:8,eta:11,phi:10"/>
+    !-- Muon system -->
```
Comment (zaborowska) [inline Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml]: missing "<"

File: Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml [hunk 2]
```diff
@@ -143,18 +123,59 @@
...
+    <constant name="Muons_Readout" type="string" value="system:4,eta:9,phi:-10"/>
```
Comment (zaborowska) [inline Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml]: This should be postfixed with "ID" (otherwise tests fail)

**Comment by jhrdinka** [comment]
Hi Anna, thank for spotting. Strangely I could still run full simulation and geoDisplay with that typo.
I will update that.

**Comment by vvolkl** [comment]
Hi Julia, looks like this still has issues. Can you please check that the calorimeter segmentation you changed is supposed to be like that?

**Comment by jhrdinka** [comment]
The problem is, that this Muon system description is still using to many system IDs. The description would need to be changed, in order to fit in.

**Comment by zaborowska** [comment]
Hi, what is the status of this PR? Do we still plan on moving the readouts to a common xml?

**Pull request closed without merging**